server/app/services/program/ProgramService.java

```diff
@@ -63,14 +63,15 @@ CompletionStage<ImmutableList<ProgramDefinition>> syncQuestionsToProgramDefiniti
       ImmutableList<ProgramDefinition> programDefinitions);
 
   /**
-   * Get the definition of a given program asynchronously.
+   * Get the definition of a given program asynchronously. The returned program may correspond to
```

Comment:
Do you mean that the id may correspond to any version? 

The returned program must be the one with this id (IIUC, one program ID corresponds to one version, let me know if that's off)

Reply:
Yeah that's a better way to phrase it, wasn't sure how to make that clear. Changed it to use your wording.